### PR TITLE
Ensure we use IMAP IDLE like it's supposed to work

### PR DESCRIPTION
**Pull request by nextgens**
## What type of PR?

enhancement

## What does this PR do?

Increase IMAP IDLE time from 2min to 29mins: this should massively help reduce network traffic & increase battery life of clients

See https://peterkieser.com/2011/03/25/androids-k-9-mail-battery-life-and-dovecots-push-imap/

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [ ] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: core/dovecot/conf/dovecot.conf [hunk 1]
```diff
@@ -100,6 +100,7 @@ service auth-worker {
 protocol imap {
   mail_plugins = $mail_plugins imap_quota imap_sieve
   mail_max_userip_connections = 20
+  imap_idle_notify_interval = 29 mins
```
Comment (Diman0) [inline core/dovecot/conf/dovecot.conf:103]: I'm not sure if it is relevant. All documentation shows no space between the value and time unit. So 
```imap_idle_notify_interval = 29mins```
I do not know if this is significant, but let's follow the dovecot practices and use no space.
https://doc.dovecot.org/settings/core/ 
shows as default `2mins` (no space)
https://doc.dovecot.org/settings/types/#time
shows 1w, weeks and  7days. (no space)

**Review by Diman0 — DISMISSED** [review]
See in line comment

File: core/dovecot/conf/dovecot.conf (hunk 1 above)
Comment (nextgens) [reply core/dovecot/conf/dovecot.conf]: I've removed it

**Review by Diman0 — APPROVED** [review]
LGTM

**Pull request merged**